addon/transitions/wormhole.js

```diff
@@ -1,3 +1,17 @@
+import $ from 'jquery';
+
+let fixPasswordInputChromeIdError = (parentElem) => {
+  let hasChildPasswordInputField = $(parentElem).find('input[type="password"]').length > 0;
+
+  if (hasChildPasswordInputField) {
+    $(parentElem).find('*').each((i, child) => {
```

Comment:
Could this be limited to just `input[type="password"]`? Seems like a lot of work to update every element's id, I'd like to scope this to be just the ids we absolutely must change to prevent the bug

Reply:
That was my initial thought but Chrome seems to complain about any other inputs and buttons in this scenario as long as they're nested in a `<form>` (i think hah). I could try to scope it to just those element types. when there is no child `<form>` i think it only complains on the actual `<input type='password'>` ...not really sure what Chrome is doing behind the scenes here

![screen shot 2018-10-16 at 3 26 20 pm](https://user-images.githubusercontent.com/666459/47041908-f8b87c00-d157-11e8-94fb-5b493430b821.png)
![screen shot 2018-10-16 at 3 26 08 pm](https://user-images.githubusercontent.com/666459/47041909-f8b87c00-d157-11e8-9405-9bb105fd907f.png)

Reply:
Ah this makes sense, it's not actually related to password inputs, it's any element that has duplicate ids. In that case, can you try to select all elements that _have_ an ID? I'm pretty sure that should be possible. If not, this solution should work